code/game/objects/effects/gibs.dm

```diff
@@ -18,36 +18,39 @@
 
 /obj/effect/gibspawner/New(location, datum/dna/MobDNA)
 	..()
+	ASSERT(length(gibtypes) == length(gibamounts))
+	ASSERT(length(gibamounts) == length(gibdirections))
```

Comment:
```suggestion
	ASSERT(length(gibtypes) == length(gibamounts) == length(gibdirections))
```
Try this

Reply:
Discussed with AA on discord and it's best to leave it as it is.